### DISCUSSION TITLE
[Announcement] HW3 is out

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs524-program-analysis#255] [Announcements]
Hi all,

This is an announcement about Homework 3.
Here is the GitHub classroom link for this homework:
https://classroom.github.com/a/_dIubkNx

## Submission
You should submit your answers in both GitHub and Gradescope.
1. Write your answer in the latex template file, and push your commit to GitHub.
2. Submit the compiled PDF file to Gradescope.

When submitting to Gradescope, **DO NOT** change the problem region defined in the Gradescope system.
Please refer to the README.md of the homework repository.

**IMPORTANT:** Write in Korean if you are a Korean student.

## Due date
Due: **4.5 (Fri.) 23:59:59**
Late Due: **4.7 (Sun.) 23:59:59** (with late submission penalty).

**Comment by leporia** [comment]
When submitted to gradescope the problem region remained empty and no pages were tagged with a question. Is this the correct behaviour?

**Comment by goodtaeeun** [comment]
@leporia I have updated the template of the assignment. You do not have to manually map questions to pages now.

**Comment by kohs100** [comment]
It seems gradescope cannot properly map answer region that goes over a page. Is this intended behaviour?

**Comment by goodtaeeun** [comment]
Sorry for the late reply. Yes, we mapped each problem to a single page.
